pkg/controller/pagerdutyintegration/clusterdeployment_created.go

```diff
@@ -73,6 +73,9 @@ func (r *ReconcilePagerDutyIntegration) handleCreate(pdclient pd.Client, pdi *pa
 	}
 
 	ClusterID := cd.Spec.ClusterName
```

Comment:
```suggestion
	clusterID := cd.Spec.ClusterName
```

This variable shouldn't be exported.

Reply:
good catch sam. thanks